src/braket/ir/jaqcd/shared_models.py

```diff
@@ -129,6 +129,21 @@ class Angle(BaseModel):
     angle: confloat(gt=float("-inf"), lt=float("inf"))
 
 
+class Probability(BaseModel):
```

Comment:
Can we change the class name to "SingleProbability" because this only accepts 1 float?